docs/IncidentsApi.md

```diff
@@ -0,0 +1,119 @@
+# \IncidentsApi
+
+All URIs are relative to *https://api.mux.com*
+
+Method | HTTP request | Description
+------------- | ------------- | -------------
+[**GetIncident**](IncidentsApi.md#GetIncident) | **Get** /data/v1/incidents/{INCIDENT_ID} | Get an Incident
+[**ListIncidents**](IncidentsApi.md#ListIncidents) | **Get** /data/v1/incidents | List Incidents
+[**ListRelatedIncidents**](IncidentsApi.md#ListRelatedIncidents) | **Get** /data/v1/incidents/{INCIDENT_ID}/related | List Related Incidents
+
+
+# **GetIncident**
+> IncidentResponse GetIncident(ctx, iNCIDENTID)
+Get an Incident
+
+Returns the details of an incident 
+
+### Required Parameters
+
+Name | Type | Description  | Notes
+------------- | ------------- | ------------- | -------------
+ **ctx** | **context.Context** | context for authentication, logging, cancellation, deadlines, tracing, etc.
+  **iNCIDENTID** | **string**| ID of the Incident | 
```

Comment:
Here we are again with the `iNCIDENT`... I hope this doesn't show up in the docs.